tests/runtime/call

```diff
@@ -657,7 +657,10 @@ TIMEOUT 3
 
 NAME map len keyless
 PROG BEGIN { @ = 0; printf("map len: %d\n", len(@)); exit(); }
-EXPECT map len: 1
+EXPECT stdin:1:40-45: ERROR: @ used as a non-scalar map, previously used as a scalar map
```

Comment:
```suggestion
EXPECT stdin:1:40-45: ERROR: @ used as a non-scalar map. 'len' only accepts maps that have keys
```

Reply:
I've updated this these messages for all calls to include the specific function and context.